### PR TITLE
Fix improper syntax with spread operator

### PR DESCRIPTION
**Pull request by mongodben**
currently the example shows incorrect JS syntax wherein an object is included within an object. 

this change fixes by using the spread operator `...`. 

it might be clearer to the user to not include `deleteDog` as a variable, but rather just put it directly into the mock results.

### Checklist:

- [ ] If this PR is a new feature, please reference an issue where a consensus about the design was reached (not necessary for small changes)
- [ ] Make sure all of the significant new logic is covered by tests

### CODE REVIEW

**Review by jpvajda — COMMENTED** [review]
Thanks for this contribution, I'm going to confirm with @benjamn  @brainkim  @hwillson if this change should be merged.

**Comment by hwillson** [comment]
Thanks for catching this @mongodben (and thanks for the ping @jpvajda). I have a feeling the intent here was to just use

```js
{
  result: { data: deleteDog },
}
```

That might be the cleanest quick change, so we don't have people thinking the spread is necessary for some reason.

File: docs/source/development-testing/testing.mdx [hunk 1]
```diff
@@ -321,7 +321,7 @@ it('should render loading state initially', () => {
         query: DELETE_DOG_MUTATION,
         variables: { name: 'Buck' },
       },
-      result: { data: { deleteDog } },
+      result: { data: { ...deleteDog } },
```
Comment (jpvajda) [inline docs/source/development-testing/testing.mdx:324]:
```suggestion
      result: { data: deleteDog },
```

File: docs/source/development-testing/testing.mdx [hunk 2]
```diff
@@ -361,7 +361,7 @@ it('should delete and give visual feedback', async () => {
...
```
Comment (jpvajda) [inline docs/source/development-testing/testing.mdx:364]:
```suggestion
      result: { data: deleteDog },
```

**Pull request merged**